server/protocols/jmap/src/main/java/org/apache/james/jmap/model/Message.java

```diff
@@ -294,7 +285,8 @@ private static boolean hasAttachment(List<Attachment> attachments) {
                                Optional<String> textBody,
                                Optional<String> htmlBody,
                                ImmutableList<Attachment> attachments,
-                               ImmutableMap<BlobId, SubMessage> attachedMessages) {
+                               ImmutableMap<BlobId, SubMessage> attachedMessages,
+                               ImmutableMap<String, Boolean> keywords) {
```

Comment:
Message is not being deserialized, right?

It is only being serialized.

Please arrage accordingly, it's duplicated informations of non exposed builder methods axposed though the constructor. Information duplication is bad.